zipline/data/minute_bars.py

```diff
@@ -552,6 +552,10 @@ def _open_minute_file(self, field, sid):
 
         return carray
 
+    def get_ctable(self, sid):
+        ctable_path = os.path.join(self._rootdir, _sid_subdir_path(sid))
```

Comment:
@ssanderson Realized the confusion here, the methods you mentioned above of the BcolzMinuteBarWriter, whereas I was adding `get_ctable` to BcolzMinuteBarReader, for consumers of the minute bars. But they probably should share this path creation logic.